easybuild/easyconfigs/h/harmony/harmony-0.1.0-20210528-foss-2020b-R-4.0.3.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'RPackage'
+
+name = 'harmony'
+local_commit = '2586afd'
+# see DESCRIPTION to determine version,
+# but also take date of last commit into account (since version isn't always bumped)
+version = '0.1.0-20210528'
```

Comment:
Just to check that the reason the version is going backwards is that the previous value was incorrect?

Reply:
The commit `2586afd` from May 28, 2021 shows version `0.1.0`:

https://github.com/immunogenomics/harmony/blob/2586afd4a25f20debd34f70e9766538e39c77b8f/DESCRIPTION#L3

However the commit `88b1e2a` from Feb 24, 2020 that was used in the previous easyconfig https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/h/harmony/harmony-1.0.0-20200224-foss-2020a-R-4.0.0.eb shows version `1.0.0`: https://github.com/immunogenomics/harmony/blob/88b1e2a1e95306cd4531eaf89ad98712af05ca99/DESCRIPTION#L4

I think this explains why the version is going backwards.